src/nav/full-stack-observability.yml

```diff
@@ -2583,13 +2583,15 @@ pages:
           - title: Enable Lambda monitoring
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring
             pages:
-              - title: 1. Account linking
+              - title: Intro to enabling Lambda monitoring
+                path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-aws-lambda-monitoring
+              - title: Step 1. Account linking
```

Comment:
@zuluecho9, how about putting the verb first to match the topic and using a colon instead of a period: "Step 1: Link accounts"?